### ISSUE TITLE
Must rerun on wake

### ISSUE THREAD

**Issue by Juice805** [AstroSnail/apexctl#7]
The extra keys on my apex 350 must be enabled on wake, is this expected? To be persistent through sleeps should I add `sudo apexctl keys on` to my `~/.xinitrc`?

**Comment by AstroSnail** [comment]
Yes, this is expected. However, this should be done automatically by udev. Have you installed the udev rules correctly?

**Comment by Juice805** [comment]
After reinstalling again the issue is now on reboot the keyboard will not accept input at all until I `sudo make uninstall`. If i then reinstall and enable custom keys they work for the remainder of the session.

**Comment by Juice805** [comment]
And my original report is mistaken. The actual issue was `sudo apexctl keys on` is not persistent after Manjaro wakes from sleep, not rebooting.

**Comment by AstroSnail** [comment]
Alright, I'll look into that to see what's going on.

**Comment by Juice805** [comment]
I'm new to a lot of linux's inner workings such as keybindings so I wasn't sure what logs would be helpful, but I'd be happy to provide any that would help.
